src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,109 @@
+package storage;
+
+import exceptions.DukeException;
+import tasks.Deadline;
+import tasks.Event;
+import tasks.TaskList;
+import tasks.ToDo;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.FileNotFoundException;
+
+/**
+ * Handles loading and saving of tasks to local storage.
+ */
+public class Storage {
+    private final Path HOME_DIRECTORY = Path.of(System.getProperty("user.dir") + "/data");
```

Comment:
I guess this variable name might be misunderstood to mean `$HOME` on unix. Would `DUKE_DATA_DIR` be better?